wasmer.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "wasmer/wasix-docs"
-version = "0.60.0"
+version = "0.84.1"
 description = "Documentation for WASIX"
 
 [dependencies]
-"sharrattj/static-web-server" = "1"
+"sharrattj/static-web-server" = "^1"
```

Comment:
This is equivalent, the change is redundant, but doesn't really matter either way.

Reply:
this was updated automatically by wasmer, so i let it be.